src/SimulationRunner.cc

```diff
@@ -538,6 +538,11 @@ void SimulationRunner::UpdateSystems()
       system->Update(this->currentInfo, this->entityCompMgr);
   }
 
+  if (serverConfig.SameProcessAsGUI())
+  {
+    this->eventMgr.Emit<events::UpdateSystems>();
+  }
```

Comment:
this will emit a signal to update `GuiSystem` plugins, the update call will update only system plugins, right ?

Reply:
Ah yes, I believe you are right :+1: this code should be okay then. Maybe we can add a comment to the code here that indicates that the event is for `GuiSystem`s so that others don't get confused with system plugins?

Reply:
I'm proposing renaming the event and added some docs in #975